homeassistant/components/delijn/sensor.py

```diff
@@ -0,0 +1,107 @@
+"""Support for De Lijn (Flemish public transport) information."""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.exceptions import PlatformNotReady
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_NEXT_PASSAGE = 'nextpassage'
+CONF_STOP_ID = 'stop_id'
+CONF_SUB_KEY = 'sub_key'
+CONF_MAX_PASSAGES = 'max_passages'
+
+DEFAULT_NAME = 'De Lijn'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_SUB_KEY): cv.string,
+    vol.Required(CONF_NEXT_PASSAGE): [{
```

Comment:
I suggest that this is renamed to `stations`.

Reply:
I don't really follow this suggestion if that's ok for you. For me stations are more related to trains, for trams and busses "stops" is the correct terminology (and it's also used like this on the official site of De Lijn: https://www.delijn.be/en/haltes/).
With Passage I mean the next passage of a bus at a stop. Which is also the correct terminology (and also here it's the terminology used on the official site of De Lijn: https://www.delijn.be/en/haltes/halte/200552/Gent_Korenmarkt_perron_2).
Hope that explains the use of this terminology.

Reply:
Can we rename this to `CONF_NEXT_DEPARTURE` which should hold `next_departure`?

Reply:
Changes made in last commit